readme.md

```diff
@@ -174,83 +192,101 @@ app.get('/users', (req, res) => {
     res.json(users);
 });
 ```
-- After adding new routes and restarting the server, visit localhost:4000/users or localhost:4000/tasks from your browser! Or simply,
-in your console type:
+
+After adding new routes and restarting the server, visit localhost:4000/users or localhost:4000/tasks from your browser! Or in your console, simply type:
+
 ```
 $ curl localhost:4000/users
```

Comment:
You should also ad the output in CURL commands. So for example:

```sh
$ curl localhost:4000/ping
{
   "pong": true
}
```

this should help the reader visualise what happens and they will know if their command is actually doing what is expected.